client/src/components/Reviews/components/ReviewList.jsx

```diff
@@ -39,6 +44,12 @@ function ReviewList({reviewsMetadata, currentFilter }) {
       .catch((err) => console.log(err));
   }, [currentSort]);
   useEffect(() => {
+    if (!reviews.results) {
+      return;
+    }
+    if (reviews.results.length % 2 !== 0) {
```

Comment:
Is this to check if we've hit the end of the reviews?

Reply:
yeah it is

Reply:
Ah ok. How do you handle the case where the total number of reviews is even?

Reply:
ah good point i didnt think of that, ill go change that